### DISCUSSION TITLE
Incorrect State

### DISCUSSION THREAD

**Discussion by MattEdwin** [aarons22/homebridge-bond#263] [Q&A]
I have everything working with my Bond and 3 ceiling fans, but I have one fan that displays an incorrect state.  In Homebridge as well as the Home app, it shows the light as "on" when it is actually "off" and the other way around.  Is there a way to correct that or at least trick it into thinking it the other state?

**Comment by fredthorne** [comment]
I have the same issue with my fan with lights on a physical switch.  Every so often my wife will switch the wrong physical switch and accidentally turn our Bond-controlled fan light off; when switching it back on, re-supplying power to the fan/light, the light will turn on though Bond/HomeKit believes it to be off.

My unsophisticated ‘trick’ has been to turn power back off to them at the physical switch, go into HomeKit and turn the light “on,” then turn the physical switch back on, resupplying power to the fan light.  Now the fan’s light, Bond, HOOBS and HomeKit are all initially on and device-state aligned, and can then be controlled using any of these connections with the fan light’s status appearing correctly.  I’ve had no trouble with the fan itself “remembering” its state. 

I’ve had no such luck getting my fireplace to behave similarly.  It’s routinely showing up as “on” on HomeKit when it’s actually off while showing as off on Bond and HOOBS. (Sigh)
